tests/api_v2/test_util_methods.py

```diff
@@ -0,0 +1,231 @@
+# Copyright (c) 2017 Shotgun Software Inc.
+# 
+# CONFIDENTIAL AND PROPRIETARY
+# 
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit 
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your 
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights 
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import os
+import sys
+
+from tank_test.tank_test_base import *
```

Comment:
Do not `import *`. This prevents linters from discovering unused variables. Only import what you need, which is `setUpModule` in this case.